src/qt/forms/debugwindow.ui

```diff
@@ -1198,13 +1198,65 @@
                 </widget>
                </item>
                <item row="6" column="0">
+                <widget class="QLabel" name="peerRelayTxesLabel">
+                 <property name="toolTip">
+                   <string>Whether the peer relays transactions: "✔" if yes, empty if no.</string>
```

Comment:
fRelay doesn't say whether the peer relays transactions. fRelay says whether the peer asked *us* to relay transactions or not.

Reply:
Thanks -- fixed